File: P1 2016.2/Questao(2)/entities/Cliente.java
package entities;

import entities.produtos.Produto;

public class Cliente {
    private String cpf;

    
    public Cliente(String cpf) {
        this.cpf = cpf;
    }

    public String getCpf() {
        return cpf;
    }
    
    public void comprar(Loja loja, Produto prod){
        loja.vender(prod);
    }
}

File: P1 2016.2/Questao(2)/python/entities/Cliente.py
from .produtos.Produto import Produto
from .Loja import Loja


class Cliente:
    def __init__(self, cpf: str) -> None:
        self.cpf = cpf

    def comprar(self, loja: Loja, prod: Produto) -> None:
        loja.vender(prod)

